src/cli/rustup_mode.rs

```diff
@@ -1598,9 +1598,15 @@ fn doc(cfg: &Cfg, m: &ArgMatches) -> Result<utils::ExitCode> {
     let doc_url = if let Some(topic) = m.get_one::<String>("topic") {
         topical_path = topical_doc::local_path(&toolchain.doc_path("").unwrap(), topic)?;
         topical_path.to_str().unwrap()
-    } else if let Some((_, _, path)) = DOCS_DATA.iter().find(|(name, _, _)| m.get_flag(name)) {
+    } else if let Some((name, _, path)) = DOCS_DATA.iter().find(|(name, _, _)| m.get_flag(name)) {
+        writeln!(
+            process().stderr().lock(),
+            "Opening docs {} in your browser",
+            name
```

Comment:
I suggest refining the wording a bit more:

```suggestion
            "Opening docs named `{name}` in your browser",
```

... since the current one might still seem a bit weird when the `name` is e.g. "rustdoc" or "unstable-book".